l10n_it_fatturapa_out/data/invoice_it_template.xml

```diff
@@ -637,7 +637,9 @@ e 'line' per riga di fattura (a seconda del livello in cui sono chiamati)
                             <ModalitaPagamento
                             t-out="record.invoice_payment_term_id.fatturapa_pm_id.code"
                         />
-                            <!--                            <DataRiferimentoTerminiPagamento t-out=""/>-->
+                            <DataRiferimentoTerminiPagamento
+                            t-out="format_date(record.invoice_date)"
```

Comment:
A livello funzionale (non il mio campo) sono sempre un po' confuso dalle date nelle fatture, in questo caso perché non `record.date`?

Reply:
Semplicemente perché è quello utilizzato per popolare `DatiGeneraliDocumento.Data` https://github.com/OCA/l10n-italy/blob/5986b189734ecfc5d964ffb3b754371b56d735ca/l10n_it_fatturapa_out/data/invoice_it_template.xml#L536

Reply:
`invoice_date` è la data del documento
`date` è la data di registrazione contabile